docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  postgres:
+    image: 'postgres:13'
+    container_name: 'java-postgres'
+    ports:
+      - 5432:5432
+    environment:
+      - 'POSTGRES_USER=postgres'
+      - 'POSTGRES_HOST_AUTH_METHOD=trust'
```

Comment:
Идея подсказывает: установи пароль))))
WARNING: POSTGRES_HOST_AUTH_METHOD has been set to "trust". This will allow
java-postgres  |          anyone with access to the Postgres port to access your database without
java-postgres  |          a password, even if POSTGRES_PASSWORD is set. See PostgreSQL
java-postgres  |          documentation about "trust":
java-postgres  |          https://www.postgresql.org/docs/current/auth-trust.html
java-postgres  |          In Docker's default configuration, this is effectively any other
java-postgres  |          container on the same system.
java-postgres  |
java-postgres  |          It is not recommended to use POSTGRES_HOST_AUTH_METHOD=trust. Replace
java-postgres  |          it with "-e POSTGRES_PASSWORD=password" instead to set a password in
java-postgres  |          "docker run".

Reply:
Целенаправленно не устанавливал пароль, чтобы облегчить авторизацию